src/ts/css/vendor/bootstrap-horizon/overrides/index.scss

```diff
@@ -0,0 +1,30 @@
+// Potential component overrides in the order they're included in ../index.scss
+@import "type";
```

Comment:
_type.scss and _forms.scss are imported here, but they are empty files?

Reply:
I removed the empty override files